packages/python2.x86.vm/tools/chocolateyinstall.ps1

```diff
@@ -9,8 +9,8 @@ try {
     packageName    = ${Env:ChocolateyPackageName}
     fileType       = 'msi'
     silentArgs     = $installArgs
```

Comment:
the 2019 test run fails with `2022/10/11 06:56:42 [python2.x86.vm] vm.common.psm1 [+] ERROR : Invalid path: C:\Python27.x86` 😕